### PR TITLE
[Docs] HA Tracker: Add minimum configuration guide.

### PR DESCRIPTION
**Pull request by suitupalex**
**What this PR does**:
Adds example minimal configuration requirements to properly enable HA Tracking.

**Which issue(s) this PR fixes**:
N/A

**Checklist**
- [ ] Tests updated
- [x] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/production/ha-pair-handling.md [hunk 1]
```diff
@@ -43,4 +43,29 @@ The replica label should be set so that the value for each prometheus is unique
 
 ### Server Side
 
-To enable handling of samples, see the [distributor flags](../configuration/arguments.md#ha-tracker) having `ha-tracker` in them.
+The minimal configuration requires:
+
+* Enabling the HA tracker code path
```
Comment (pracucci) [inline docs/production/ha-pair-handling.md:48]: I would be more specific. I would also omit the "code path"

```suggestion
* Enabling the HA tracker via `-distributor.ha-tracker.enable=true` CLI flag (or its YAML config option)
```

File: docs/production/ha-pair-handling.md [hunk 2]
```diff
@@ -43,4 +43,29 @@ The replica label should be set so that the value for each prometheus is unique
...
+* Configuring the KV store for the ring (See: [Ring/HA Tracker Store](https://cortexmetrics.io/docs/configuration/arguments/#ringha-tracker-store))
```
Comment (pracucci) [inline docs/production/ha-pair-handling.md:49]: Please use internal links, so:

```suggestion
* Configuring the KV store for the ring (See: [Ring/HA Tracker Store](../configuration/arguments.md#ringha-tracker-store))
```

File: docs/production/ha-pair-handling.md [hunk 3]
```diff
@@ -43,4 +43,29 @@ The replica label should be set so that the value for each prometheus is unique
...
+* Setting the limits configuration to accept samples
```
Comment (pracucci) [inline docs/production/ha-pair-handling.md:50]:
```suggestion
* Setting the limits configuration to accept samples via `-distributor.ha-tracker.enable-for-all-users` (or its YAML config option)
```

File: docs/production/ha-pair-handling.md [hunk 4]
```diff
@@ -43,4 +43,29 @@ The replica label should be set so that the value for each prometheus is unique
...
+
+```yaml
```
Comment (pracucci) [inline docs/production/ha-pair-handling.md:52]:
```suggestion

The following configuration snippet shows an example of the HA tracker config via YAML config file:

```yaml
```

File: docs/production/ha-pair-handling.md [hunk 5]
```diff
@@ -43,4 +43,29 @@ The replica label should be set so that the value for each prometheus is unique
...
+    enable_ha_tracker: true
+    ...
+    kvstore:
+      [store: <string> | default = "consul"]
+      [consul | etcd | memberlist: <config>]
+      ...
+  ...
+```
+
+For further configuration file documentation, see the [distributor section](https://cortexmetrics.io/docs/configuration/configuration-file/#distributor_config) and [Ring/HA Tracker Store](https://cortexmetrics.io/docs/configuration/arguments/#ringha-tracker-store).
```
Comment (pracucci) [inline docs/production/ha-pair-handling.md:69]:
```suggestion
For further configuration file documentation, see the [distributor section](../configuration/config-file-reference.md#distributor_config) and [Ring/HA Tracker Store](../configuration/arguments.md#ringha-tracker-store).
```

**Comment by suitupalex** [comment]
Reupped!

**Pull request merged**